ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -97,6 +97,7 @@ Assign Vars According To Product
     IF    "${PRODUCT}" == "RHODS"
         Set Suite Variable    ${OPERATOR_APPNAME}     "Red Hat OpenShift AI"
         Set Suite Variable    ${OPERATOR_DEPLOYMENT_NAME}    rhods-operator
+        Set Suite Variable    ${OPERATOR_NAMESPACE}    redhat-ods-operator
```

Comment:
This variable is already in the test-variables.yaml file, there shouldn't be the need of adding it here

Reply:
Removing